edx_rest_api_client/auth.py

```diff
@@ -17,18 +17,19 @@ def __init__(self, username, full_name, email, signing_key, issuer=None, expires
         self.tracking_context = tracking_context
 
     def __call__(self, r):
-
+        now = datetime.datetime.utcnow()
         data = {
             'username': self.username,
             'full_name': self.full_name,
             'email': self.email,
+            'iat': now,
```

Comment:
Thanks for the clarification on the expected behavior. That means that if the expire time is set to 0 for any reason, we'll use 0 to mean don't expire.